syz-manager/manager.go

```diff
@@ -664,6 +664,10 @@ func (mgr *Manager) loadCorpus() {
 		// Version 3->4: to shake things up.
 		flags &= ^fuzzer.ProgMinimized
 		fallthrough
+	case 4:
+		// Version 4->5: fix for comparison argument sigh extension.
```

Comment:
Nit: sigh -> sign?

And in general: do we really want to do it this way (once and for all corpus)? We won't finish all these tens of thousands smash jobs in a single instance uptime.

Also:
* We lose and re-discover some share of the corpus every restart. So we will rerun hints for many parts of the kernel anyway (eventually).
* Maybe we could explicitly drop `ProgSmashed` and `ProgMinimized` for some small subset of the corpus every restart? Or, once we're ready to really update the db format (and not just bump the version), we could remember the last time we actually **finished** the smash job for every corpus item.

Reply:
Fixed the comment.

Re re-smashing. This is the current way to doing this. If we do some better mechanism,  let's do separately.
I would assume that the ones we are dropping on each restart, are the least interesting ones. I don't think they always rotate, so that eventually we drop and resmash whole corpus. I think the ones we don't drop are the most interesting and useful ones.